Examples/clipboard-demo.lisp

```diff
@@ -0,0 +1,101 @@
+(defpackage :clipboard-demo
```

Comment:
this demo needs some work, here is a screenshot:
![image](https://user-images.githubusercontent.com/6871201/54228243-05bc7a00-4502-11e9-8ade-9763c2ae032d.png)

optimally inspector is smaller, wrapping is made by word and whole frame has bigger width (so there is no need to scroll). moreover paste should be performed in a separate pane than instructions for clarity. alternatively a horizontal line should be drawn between instructions and the are where paste is put.

Reply:
I have made changes to the example code according to these recommendations. Please take a look and let me know if it's better.

Reply:
this looks much better. please also add command to clear "paste pane". moreover "menu" lacks "copy presentation to clipboard".